Lib/fontTools/t1Lib/__init__.py

```diff
@@ -108,11 +108,12 @@ def parse(self):
 
 def read(path, onlyHeader=False):
 	"""reads any Type 1 font file, returns raw data"""
-	normpath = path.lower()
+	_, ext = os.path.splitext(path)
```

Comment:
It looks like splitext doesn’t work with PathLike objects on python 3.5
You may have to convert to str/bytes first, then splitext

I’d love to drop 3.5 support and require 3.6+ (but let’s discuss this in the appropriate issue)

Reply:
I guess the fspath thing has to be done everywhere where a str is expected. Or: refactor everything to work on paths, as a long-term solution.

Reply:
I'm tempted to skip the tests below Python 3.6, somehow I can't get it to work in 3.5.

Reply:
that makes sense